decidim-core/app/controllers/decidim/messaging/conversations_controller.rb

```diff
@@ -14,19 +14,19 @@ class ConversationsController < Decidim::ApplicationController
       helper_method :username_list, :conversation
 
       def new
-        enforce_permission_to :create, :conversation
         @form = form(ConversationForm).from_params(params)
+        conversation = conversation_between(current_user, @form.recipient)
 
-        redirect_back(fallback_location: profile_path(current_user.nickname)) && return unless @form.recipient
+        return redirect_back(fallback_location: profile_path(current_user.nickname)) && return unless @form.recipient
```

Comment:
don't we need to give some feedback to the user when not being ale to create a conversation?
Otherwise I'm missing something what's this use case? can you document it via a comment, for example?

Reply:
I've documented the method. The fact is that never has been any notification (this method hasn't really changed) here if someone is trying to send a message to a non-existing user.  If you think we should add a flash error, I'll add it.